index.html

```diff
@@ -4,9 +4,107 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700;400&family=Raleway:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/axit_css/normalize.css">
+    <link rel="stylesheet" href="assets/axit_css/body.css">
+    <link rel="stylesheet" href="assets/axit_css/header.css">
+    <link rel="stylesheet" href="assets/axit_css/banner.css">
+    <link rel="stylesheet" href="assets/axit_css/social.css">
+    <link rel="stylesheet" href="assets/axit_css/tabs.css">
+    <title>AXIT</title>
```

Comment:
try add one import css
read about import in CSS 
if you don't understand how, I will help